docs/notebooks/image_to_image_arcface.ipynb

```diff
@@ -0,0 +1,362 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "p8jc8EyfruKw"
+      },
+      "source": [
+        "# Image-to-Image Search with ArcFaceLoss\n",
+        "\n",
+        "<a href=\"https://colab.research.google.com/drive/1ZS9FmnR9FzO_JYGCPazFM7TcMNQl51xM?usp=sharing\"><img alt=\"Open In Colab\" src=\"https://colab.research.google.com/assets/colab-badge.svg\"></a>\n",
+        "\n",
+        "Searching visually similar images with image queries is a very popular use case. However, using pre-trained models does not deliver the best results – the models are trained on general data that lack the particularities of your specific task. Here's where Finetuner comes in! It enables you to accomplish this easily.\n",
+        "\n",
+        "This guide will demonstrate how to fine-tune a ResNet model using the `ArcFaceLoss` function.\n",
+        "\n",
+        "*Note, please consider switching to GPU/TPU Runtime for faster inference.*\n",
```

Comment:
```suggestion
        "*Note, please switch to a GPU/TPU Runtime or this will be extremely slow!*\n",
```

It needs stronger language. This isn't just advice.